app/policies/application_policy.rb

```diff
@@ -26,15 +22,15 @@ def create?
   end
 
   def new?
-    create?
+    user.admin?
   end
 
   def update?
     user.admin?
   end
 
   def edit?
-    update?
+    user.admin?
   end
```

Comment:
These changes also seem out of place.

Reply:
It also means that the UserPolicy will return false for `edit?`, because it only overrides `update?`

Reply:
Well spotted. These changes were not intentional.